### DISCUSSION TITLE
OverwriteFlux function in Flux_Register class

### DISCUSSION THREAD

**Discussion by wjwhit** [AMReX-Codes/amrex#3214] [Q&A] [answered]
I looked at the OverwriteFlux function in Flux_Register class. I think it is correcting the coarse flux with the fine flux, but I could not find any comment for this function. Does anyone can confirm the purpose of the function. Thanks!

**Comment by WeiqunZhang** [comment, marked as answer]
It overwrites the coarse flux at the coarse/fine interface (and the interface only) with the fine flux stored in the FluxRegister.  It was provided for a specific application.

**Reply by WeiqunZhang** [reply]
#3216
